priam/src/main/java/com/netflix/priam/utils/TokenManager.java

```diff
@@ -45,9 +72,10 @@
          * unit test failures.
          */
         Preconditions.checkArgument(position >= 0, "position must be >= 0");
-        return MAXIMUM_TOKEN.divide(BigInteger.valueOf(size))
+        return tokenRangeSize.divide(BigInteger.valueOf(size))
```

Comment:
Ok, I agree, RandomPartitioner, tokenRangeSize == MAXIMUM_TOKEN base on the following logic:
 MINIMUM_TOKEN_RANDOM = BigInteger.ZERO;  //noop for non Murmur partitioner.
minimumToken= MINIMUM_TOKEN_RANDOM.add(BigInteger.ZERO); //noop for non Murmur partitioner (i.e. minimumtokekn = 0)

MAXIMUM_TOKEN_RANDOM = new BigInteger("2").pow(127);  //original logic applicable to Murmur and non Murmur partitioner.
maximumToken = MAXIMUM_TOKEN_RANDOM.add(BigInteger.ZERO); //noop for non Murmur partitioner (i.e. maximumToken equates to MAXIMUM_TOKEN_RANDOM

tokenRangeSize = maximumToken.subtract(minimumToken);  //noop for non Murmur partitioner (i.e. tokenRangeSize equates to MAXIMUM_TOKEN_RANDOM)